In [1]:
import pandas as pd

In [2]:
df =pd.read_csv("Placement.csv")

In [7]:
df.cov()

sl_no         ssc_p         hsc_p      degree_p  \
sl_no       3870.000000    -52.641355    -58.106028    -40.413645   
ssc_p        -52.641355    117.228377     60.348373     42.897137   
hsc_p        -58.106028     60.348373    118.755706     34.819820   
degree_p     -40.413645     42.897137     34.819820     54.151103   
etest_p       52.556168     37.659225     35.461678     21.929469   
mba_p          8.102336     24.535952     22.555846     17.272020   
salary    361617.668689  28777.386468  66977.716032 -11739.948520   

                etest_p         mba_p        salary  
sl_no         52.556168      8.102336  3.616177e+05  
ssc_p         37.659225     24.535952  2.877739e+04  
hsc_p         35.461678     22.555846  6.697772e+04  
degree_p      21.929469     17.272020 -1.173995e+04  
etest_p      176.251018     16.886973  2.287876e+05  
mba_p         16.886973     34.028376  9.624979e+04  
salary    228787.619507  96249.789024  8.734295e+09

1.Covariance between degree_p and etest_p:

The covariance between degree_p and etest_p is approximately 21.929469. This positive covariance indicates that degree_p and etest_p tend to vary together in the same direction.

2.Covariance between etest_p and mba_p:

The covariance between etest_p and mba_p is approximately 16.89.
This positive covariance indicates that as etest_p increases, mba_p tends to increase as well, showing a tendency for both variables to vary together in the same direction.

In [6]:
df.corr()

sl_no     ssc_p     hsc_p  degree_p   etest_p     mba_p    salary
sl_no     1.000000 -0.078155 -0.085711 -0.088281  0.063636  0.022327  0.063764
ssc_p    -0.078155  1.000000  0.511472  0.538404  0.261993  0.388478  0.035330
hsc_p    -0.085711  0.511472  1.000000  0.434206  0.245113  0.354823  0.076819
degree_p -0.088281  0.538404  0.434206  1.000000  0.224470  0.402364 -0.019272
etest_p   0.063636  0.261993  0.245113  0.224470  1.000000  0.218055  0.178307
mba_p     0.022327  0.388478  0.354823  0.402364  0.218055  1.000000  0.175013
salary    0.063764  0.035330  0.076819 -0.019272  0.178307  0.175013  1.000000

Correlation between mba_p and salary:

The correlation coefficient between mba_p and salary is approximately 0.175013.

This positive correlation suggests a slight tendency for higher MBA percentage (mba_p) to be associated with higher salaries (salary), indicating a weak relationship between the two variables.